tools/initialize.py

```diff
@@ -0,0 +1,404 @@
+import os
+import sys
+import math
+import numpy as np
+import torch
+import torch.distributed as dist
+import torch.multiprocessing as mp
+
+from timeit import default_timer as timer
+from datetime import timedelta
+
+from utils import augment_node_data, augment_edge_data,\
+                  read_nodes_file, read_edges_file,\
+                  read_node_features_file, read_edge_features_file,\
+                  read_metis_partitions, read_json,\
+                  get_node_types, write_metadata_json
+from globalids import assign_globalids_nodes, assign_globalids_edges,\
+                      get_globalids_edges
+from gloo_wrapper import gather_metadata_json
+from convert_partition import gen_dgl_objs
+
+def send_node_data(rank, node_data, part_list):
```

Comment:
why are the functions for sending and receiving data in a file named `initialize.py`?